### DISCUSSION TITLE
Discord community call: 25 May 2024

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#6801] [Support] [answered]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: May 25th, 2024 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/may-25-2024/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- @joostdecock may or may not be on a top secret black ops mission behind enemy lines for the next 3 weeks, but in any case he will essentially drop off the internet during this time and not be reachable. Until (if?) he resurfaces, @woutervdub is interim-CEO of FreeSewing.
- TBD

**Comment by woutervdub** [comment]
I hope I can make it. I'll be somewhere on the road at that time.

**Reply by BenJamesBen** [reply, marked as answer]
### May 25, 2024 Community Call

#### Notes:
- If you don't include seam allowance on a pattern, then how do you construct the garment?
- Various seam allowance widths in patterns and commercial patterns.
- Complaints about commercial patterns, how the sizing/grading doesn't reflect reality.
- New FreeSewing trousers designs in development. Jodhpurs-style trousers for riding.
- Getting trousers and shirt patterns to fit different body shapes.
- Advantages of raglan-style shirts for fitting various body areas.

#### Attendees:
- Benjamin F.
- bobgeorgethe3rd
- Curvalacia (a.k.a. Anna Puk)
- RockerKitten
- TangerineShark
- Thrunic
- Vili
